Server/mods/deathmatch/logic/CResourceMapItem.h

```diff
@@ -10,8 +10,6 @@
  *****************************************************************************/
 #pragma once
 
-#include "CResourceFile.h"
-
```

Comment:
The class in this file inherits from that base class. Why do you want to remove the include for it here?

Reply:
it compiles, i suspect it is included in different file

Reply:
IMHO a header must include everything it needs to "stand on its own". A precompiled header is just a tool to help with compilation times and, again my own opinion, headers should not be written in a style where such precompiled header exists.

Reply:
i didn't say it coming from stdinc.h

Reply:
Yes, but that's not the problem I see with it.

Reply:
what's the problem? how can i solve it?